lib/session.ex

```diff
@@ -101,6 +101,9 @@ defmodule Janus.Session do
 
   @impl true
   def init({connection, timeout}) do
+    Process.flag(:trap_exit, true)
+    Process.link(connection)
```

Comment:
It's not a valid solution: link is bidirectional, so stopping a session will shut down the connection and since we assume there can be many sessions on one connection - we want to avoid that.
Besides, trapping exit just to stop the process makes no sense - the process would exit anyway
You should take a look at https://hexdocs.pm/elixir/Process.html#monitor/1

Reply:
So, practically it will look almost the same but instead of linking `Process.monitor` will be called and accordingly `handle_info` callback will be changed, no trapping involved?

Reply:
Yes